website/blog/2023-08-14-aa-compliance-accessibility.md

```diff
@@ -0,0 +1,145 @@
+---
+title: "AA Compliant Website Accessibility through Wavemaker Platform"
+author: Bhargavi Gunda
+---
+
+On a low-code platform, achieving AA compliance requires a concerted effort to implement various accessibility standards and best practices. By following WCAG guidelines, using semantic markup, prioritizing keyboard accessibility, ensuring color contrast, providing alternative text for images, and implementing ARIA attributes, developers can create an inclusive and accessible platform for users of all abilities. Continuous testing, documentation, and training further contribute to the platform's commitment to accessibility and enable a seamless and enriching experience for all users.
+
+In the digital age, inclusivity and accessibility are no longer just buzzwords; they are crucial pillars of a successful online presence. Understanding the diverse needs of our audience, we embarked on a transformative journey to ensure our low-code platform is not just AA compliant but also embraces the power of semantic/native markup.
+Achieving AA compliance on a low-code platform requires careful consideration of various accessibility standards and guidelines. 
+
+Although Accessibility is designed for people with disabilities, it helps everyone in general. Accessibility promotes usability. Everyone, in general, can benefit from clear instructions, opportunities to correct form errors, simple visual layouts, high color contrast, and the option to read a transcript or captions to a video or audio recording.
+Accessibility often describes hardware and software designed to help those who experience disabilities.
+
+<!-- truncate -->
+
+## Standard practices to ensure AA compliance
+
+**WCAG 2.1 Guidelines**: The Web Content Accessibility Guidelines (WCAG) 2.1 provide the most widely recognized and accepted accessibility standards. These guidelines define criteria for making web content more accessible to users with disabilities.
+
+**Keyboard Accessibility**: All components and interactions should be operable and fully functional using only the keyboard. This ensures that users who rely on keyboard navigation or assistive devices can access all functionalities without barriers.
+Focus Management: Proper focus management is crucial for keyboard accessibility. Ensuring that focus is clearly visible and moves in a logical order through interactive elements improves the overall accessibility of the platform.
```

Comment:
What about focus trap inside the modal dialogs? We should talk about that too...?